cmplibrary/src/test/java/com/sourcepoint/cmplibrary/ConsentLibBuilderTest.java

```diff
@@ -0,0 +1,184 @@
+package com.sourcepoint.cmplibrary;
+
+import android.app.Activity;
+import android.view.ViewGroup;
+
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ConsentLibBuilderTest {
+
+
+    private ConsentLibBuilder consentLibBuilder;
+    private ConsentLib.Callback callback;
+
+    @Before
+    public void initConsentLibBuilder() {
+        consentLibBuilder = new ConsentLibBuilder(808, "cybage.com", mock(Activity.class));
+        callback = new ConsentLib.Callback() {
+            @Override
+            public void run(ConsentLib c) {
+
+            }
+        };
+    }
+
+    public Field getDeclaredFieldAccess(String fieldName) throws Exception{
+        Field member = ConsentLibBuilder.class.getDeclaredField(fieldName);
+        member.setAccessible(true);
+        return member;
+    }
+
+    @Test(expected = ConsentLibException.class)
+    public void testBuild() throws ConsentLibException {
+        ConsentLib consentLib = mock(ConsentLib.class);
+        assertEquals(consentLib, consentLibBuilder.build());
+    }
+
+    @Test
+    public void testSetMessageTimeout() {
+        consentLibBuilder.setMessageTimeOut(20000);
```

Comment:
I've checked a few other [testing examples](https://github.com/googlesamples/android-testing/blob/master/runner/AndroidJunitRunnerSample/app/src/androidTest/java/com/example/android/testing/androidjunitrunnersample/CalculatorTest.java) and I believe we have to rename the method names to something related to what's being tested in that method.  For example:
Instead of 
```java
 @Test
    public void testSetMessageTimeout() 
```
We should name it 
```java
 @Test
    public void setMessageTimeout() {
```

Reply:
I would rename the methods by removing "test" prefix.